source/_integrations/sun.markdown

```diff
@@ -9,6 +9,7 @@ ha_codeowners:
   - '@Swamp-Ig'
 ha_iot_class: Calculated
 ha_domain: sun
+ha_config_flow: true
 ---
 
 The sun integration will use your current location to track if the sun is above or
```

Comment:
We should rephrase, the integration will not use "your current location", it will use the location of the Home Assistant server/installation.

Reply:
Adjusted.